crates/sui-graphql-rpc/src/types/validator_credentials.rs

```diff
@@ -4,6 +4,7 @@
 use super::base64::Base64;
 use async_graphql::*;
 
+/// The credentials related fields associated with a validator.
```

Comment:
Same here, not sure how useful this comment would be

Reply:
I updated it, hope it will help more phrased like this.